src/public/utils.js

```diff
@@ -78,8 +82,7 @@ export function utils_fmt_history_type(val) {
 
 
 // -------------- config functions --------------------
-export function utils_config_items($w, data) {
-	let config = g_hist_map;
```

Comment:
right, nice, I see it is not specific anymore, so disconsider my message about renaming the function